LoginApp/app/src/main/java/com/example/loginapp/localDb/RealmUserDetailsOperation.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.loginapp.localDb
+
+import com.example.loginapp.entity.UserDetails
+import io.realm.Realm
+import io.realm.RealmConfiguration
+
+object RealmUserDetailsOperation {
+
+    private val realmInstance : Realm = Realm.getInstance(RealmConfig.getRealmConfiguration() as RealmConfiguration)
+    private val EMAIL_FIELD : String = "email"
+    private val PASSWORD_FIELD : String = "password"
+
+    fun writeUserDetailsToRealm(
```

Comment:
@NayakRoshan what would happen if this function is called from different thread?

Reply:
@rishabh-jain-1990 Realm would throw an error as the realm instance was instantiated in the ui thread. So I will have to create a realm instance in that thread itself. Could you please tell me why realm doesnt allow this?

Reply:
So I will create a realmInstance in that specific thread and pass it as an instance to the method. Is that a good practice?